packaging/windows/sharedframework/registrykeys.wxs

```diff
@@ -13,6 +13,26 @@
           <RegistryValue Action="write" Name="1.0.0-rc2" Type="integer" Value="1" KeyPath="yes"/>
         </RegistryKey>
       </Component>
+
+<!-- Set registry keys to allow WER to genereate correct dumps-->
+      <Component Directory="TARGETDIR">
+        <RegistryKey Root="HKLM" Key="SOFTWARE\Microsoft\Windows\Windows Error Reporting\RuntimeExceptionHelperModules">
+          <RegistryValue Action="write" Name="[DOTNETHOME]shared\$(var.FrameworkName)\$(var.FrameworkDisplayVersion)\mscordaccore.dll" Type="integer" Value="0" KeyPath="yes"/>
```

Comment:
1. this is "1.1.0-preview1-001104-00" part. 
2. I didn't find if I can get a full path without constructing it, but I'm not familiar with the WIX tool nor with the code base. So if there is any way to get them, I'll change to use it.